src/main/java/com/ncc/aif/ValidateAIF.java

```diff
@@ -287,6 +303,18 @@ private static boolean processArgs(String[] args, Set<ArgumentFlags> flags, Set<
             logger.error("Please specify either file(s) or a directory of files to validate.");
             return false;
         }
+        if (abortStr != null) {
+            try {
+                // TODO: separate command-line validator from validator class
+                abortParam = Integer.parseUnsignedInt(abortStr);
+            } catch (NumberFormatException nfe) {
+                abortParam = -1;
+            }
+            if (abortParam < 1) {
```

Comment:
Should we also prevent 1 from being allowed? I know you have something in the README, but as it stands, 1 isn't really valid. What do you think?

Reply:
Hmm, I can probably argue both sides.  But it was probably re-introduced when we (or maybe just I) thought that failfast was only going to be in a "developer edition" of AIF, so we could be a little more lax about these kinds of things, since basically only AIF developers (i.e., NCC) would be using it, almost as a sandbox.  But if master is always our "best version," then I agree that num=1 should probably not be allowed.

I'll change to code to reject it on the command line, but perhaps still allow it via programmatic access (e.g., `setAbortThreshold()`) and remove the NOTE from the README.

Reply:
By the way, I've also seen an abort threshold of 2 erroneously return a valid KB, so I've set the minimum (and default) to 3.